cmd/issue_edit_test.go

```diff
@@ -0,0 +1,234 @@
+package cmd
+
+import (
+	"log"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/spf13/pflag"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+// create an issue and return the issue number
+func issueEditCmdTest_createIssue(dir string) string {
+	cmd := exec.Command("../lab_bin", "issue", "create", "lab-testing",
+		"-m", "issue title", "-l", "bug")
+	cmd.Dir = dir
+
+	b, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	s := strings.Split(string(b), "\n")
+	s = strings.Split(s[0], "/")
+	return s[len(s)-1]
+}
+
+func issueEditCmdTest_showIssue(dir string, issueNum string) string {
+	cmd := exec.Command("../lab_bin", "issue", "show", "lab-testing", issueNum)
+	cmd.Dir = dir
+
+	b, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
consider passing the `*testing.T` to these helper function so you can do a `t.Fatal(err)` here instead of the `log.Fatal(err)`